### PR TITLE
Fix comments about when to pivot_root

### PR DESCRIPTION
**Pull request by hqhq**
Signed-off-by: Qiang Huang <h.huangqiang@huawei.com>

### CODE REVIEW

File: libcontainer/rootfs_linux.go [hunk 1]
```diff
@@ -37,8 +37,7 @@ func needsSetupDev(config *configs.Config) bool {
 }
 
 // prepareRootfs sets up the devices, mount points, and filesystems for use
-// inside a new mount namespace. It doesn't set anything as ro or pivot_root,
-// because console setup happens inside the caller. You must call
+// inside a new mount namespace. It doesn't set anything as ro. You must call
```
Comment (cyphar) [inline libcontainer/rootfs_linux.go]: Can you finish this sentence (I forgot to finish it in the original patch). It should read something like `You must call finalizeRootfs after this function to finish setting up the rootfs`. Thanks.

Comment (TomSweeneyRedHat) [inline libcontainer/rootfs_linux.go]: 👍

**Comment by hqhq** [comment]
@cyphar Done, thanks.

**Comment by cyphar** [comment]
LGTM. Cheers.

**Comment by crosbymichael** [comment]
LGTM

**Pull request merged**